systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeST.java

```diff
@@ -67,19 +66,14 @@ class HttpBridgeST extends AbstractST {
 
     private final String httpBridgeClusterName = "http-bridge-cluster-name";
 
-    private String kafkaClientsPodName;
-
     @ParallelTest
     void testSendSimpleMessage(ExtensionContext extensionContext) {
-        final TestStorage testStorage = new TestStorage(extensionContext);
-        final String topicName = testStorage.getTopicName();
-        final String producerName = "producer-" + new Random().nextInt(Integer.MAX_VALUE);
-        final String consumerName = "consumer-" + new Random().nextInt(Integer.MAX_VALUE);
+        final TestStorage testStorage = storageMap.get(extensionContext);
```

Comment:
You have it all places, but why it's better to use `storageMap` than `new TestStorage()`? I think that ultimately you should not have access to `storageMap` directly in the tests and you should have everything generated based on the `extenstionContext`. IIRC we wanted to remove `storageMap` and use `new TestStorage`.

Reply:
I based this upon the storageMap logic, as its a beforeEach thing, where the testStorage instance is being put into the map, but if you say there will be a storageMap removal soon, then these lines of this refactor make no sense. I should rewrite these lines to be rather new obj init instead. Thanks !